recipes-devtools/vpi/libnvvpi2_2.1.6.bb

```diff
@@ -42,6 +42,22 @@ do_install() {
     ln -snf lib/aarch64-linux-gnu ${D}/opt/nvidia/vpi2/lib64
 }
 
+pkg_postinst:${PN}() {
+    if [ -e "$D/opt/nvidia/vpi2/etc/ld.so.conf.d/vpi2.conf" ]; then
+        if [ ! -d "$D${sysconfdir}/ld.so.conf.d" ]; then
+            install -d $D${sysconfdir}/ld.so.conf.d
+        fi
+        ln -sf /opt/nvidia/vpi2/etc/ld.so.conf.d/vpi2.conf $D${sysconfdir}/ld.so.conf.d/
+    fi
+    if [ ! -d "$D/lib/firmware" ]; then
```

Comment:
`/lib` should be `${nonarch_base_libdir}` here.

Reply:
I'm doing this  already